Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -103,7 +103,9 @@ private extension BrowserToTrayAnimator {
 
         // Force subview layout on the collection view so we can calculate the correct end frame for the animation
         tabTray.view.layoutSubviews()
-        tabTray.collectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: bvc.tabManager.selectedIndex, inSection: 0), atScrollPosition: .CenteredVertically, animated: false)
+        if tabTray.collectionView.numberOfItemsInSection(0) > bvc.tabManager.selectedIndex {
+            tabTray.collectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: bvc.tabManager.selectedIndex, inSection: 0), atScrollPosition: .CenteredVertically, animated: false)
+        }
```

Comment:
I need more context about this change. What does this fix? Shouldn't `tabTray.collectionView.numberOfItemsInSection(0)` always be greater than `bvc.tabManager.selectedIndex`?